examples/dgmulti_2d/elixir_advection_diffusion.jl

```diff
@@ -11,48 +11,52 @@ initial_condition_zero(x, t, equations::LinearScalarAdvectionEquation2D) = SVect
 initial_condition = initial_condition_zero
 
 # tag different boundary segments
-left(x, tol=50*eps()) = abs(x[1] + 1) < tol
-right(x, tol=50*eps()) = abs(x[1] - 1) < tol
-bottom(x, tol=50*eps()) = abs(x[2] + 1) < tol
-top(x, tol=50*eps()) = abs(x[2] - 1) < tol
+left(x, tol = 50 * eps()) = abs(x[1] + 1) < tol
+right(x, tol = 50 * eps()) = abs(x[1] - 1) < tol
+bottom(x, tol = 50 * eps()) = abs(x[2] + 1) < tol
+top(x, tol = 50 * eps()) = abs(x[2] - 1) < tol
 is_on_boundary = Dict(:left => left, :right => right, :top => top, :bottom => bottom)
 
 cells_per_dimension = (16, 16)
 mesh = DGMultiMesh(dg, cells_per_dimension; is_on_boundary)
 
 # BC types
-boundary_condition_left = BoundaryConditionDirichlet((x, t, equations) -> SVector(1 + 0.1 * x[2]))
+boundary_condition_left = BoundaryConditionDirichlet((x, t, equations) -> SVector(1 +
+                                                                                  0.1 *
+                                                                                  x[2]))
```

Comment:
This looks odd having 3 extra line breaks for `SVector`

Reply:
It's too long if we tried to put it in one line. We would have to use shorter variable names if we wanted to achieve that.